src/hypersync/IDataSource.ts

```diff
@@ -12,6 +12,7 @@ export interface IDataSetResultComplete<TData = DataObject> {
   data: TData;
   apiUrl: string;
   context?: TokenContext;
+  nextPage?: string;
```

Comment:
Nit: Seems like `nextPage` should come before `context` since it is "closer to the data".